.github/workflows/probot-check-group.yml

```diff
@@ -12,12 +12,14 @@ jobs:
   required-jobs:
     runs-on: ubuntu-latest
     if: github.event.pull_request.draft == false
-    # if this timeout triggers, then the job needs to be manually restarted through the GitHub interface
-    timeout-minutes: 60
+    timeout-minutes: 61  # in case something is wrong with the internal timeout
```

Comment:
why 61 instead of 60?

Reply:
ok, found it, but just in case make it 65?

Reply:
1 minute is plenty. Computers are really fast